child_compassion/models/child_compassion.py

```diff
@@ -349,6 +349,18 @@ def _compute_available(self):
         for child in self:
             child.is_available = child.state in self._available_states()
 
+    @api.depends("pictures_ids")
+    def _compute_portrait(self):
+        for child in self:
+            if child.pictures_ids:
+                child.portrait = child.pictures_ids.sorted()[0].headshot
```

Comment:
Like this it doesn't fail when no pictures exist.
```suggestion
                child.portrait = child.pictures_ids.sorted()[:1].headshot
```

Reply:
From what I've seen, the check if `child.picture_ids` should already test that the list is not empty or `None` (I had to [look it up](https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty) because I'm still not used to python), so it should not fail. 

Do you still think that it's better to index with `[:1]` ? I personally feel that it's clearer with `[0]` but it might just be that I'm not used to it.

Reply:
`child.pictures_ids` can be an empty recordset which will evaluate to False. But accessing an element in an empty set with an index is not allowed and will raise and IndexError. However, slicing the set will work and return a subset (empty in this case), and then `.headshot` will return False without raising an error.